### PR TITLE
CDH-3903: Initial version of release notes

### PR DESCRIPTION
**Pull request by cheinz-sag**

### CODE REVIEW

File: content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Improved handling of additional columns
+
+The UI simplifies duplicating an additional column and changing the data type of the column.
+
+### Limitations
+
+|<div style="width:250px">Description</div>
+|:---
+|Mixed usage of uppercase and lowercase characters for attribute names in the documents is not supported.|
```
Comment (doernemt) [inline content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md:29]: This is not the case anymore. Dremio now can read both lowercase and uppercase properties from MongoDB. I.e., myprop and myProp would now both be read (as myprop). The only remaining limitation is that if BOTH attributes are set for the same document, only one of the values would be visible.

File: content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Cumulocity IoT DataHub Release 10.18 includes the following improvements, limitations, and known issues:
+
+### New version of internal query engine
+
+Cumulocity IoT DataHub now uses version 24 of Dremio as query engine. The new version comes along with various features and improvements like a revamped user interface, simplified interaction with PowerBI, improved query performance.
+
+### Efficient ODBC connectivity
+
+The ODBC driver required for running queries against Dremio now relies on Apache Arrow, whose columnar memory format allows for a more efficient ODBC-based query execution.
```
Comment (doernemt) [inline content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md:15]: Would add that the legavy driver continues to work and that the new driver will use a different port (32010 by default)

**Review by doernemt — APPROVED** [review]

File: content/release-10-18-0/announcements-10-18-0.md [hunk 3]
```diff
@@ -201,3 +201,26 @@ For more details, see the [API Reference for EPL (ApamaDoc)](https://documentati
...
+
+#### Planned
+
+##### Breaking change in the offloading mechanism - switch to new table format may introduce incompatibilities
+
+In future releases of Cumulocity IoT DataHub the offloading mechanism may leverage the Apache Iceberg table format, which is an open format supported by Dremio. Due to this change, current offloading configurations may fail. For example, they may fail due to rarely used data types which are incompatible with the Apache Iceberg format. Further details and remediation actions will be provided closer to the respective release.
+
+##### Breaking change in the offloading mechanism - removal of history column in alarms collection
+
+In a future release of Cumulocity IoT core the history attribute will be removed from the alarms. Therefore, Cumulocity IoT DataHub does no longer include this column in newly defined offloadings. If alarms data has been already offloaded into the data lake, the Dremio table associated with that data lake table still includes the history column, but for new offloading runs the value will not be included anymore and thus will be null.
```
Comment (MWindel) [inline content/release-10-18-0/announcements-10-18-0.md:216]: null = zero?

File: content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+|When using Microsoft Azure as data lake with IP white-listing being activated, Cumulocity IoT DataHub cannot access the data lake.|
+
+### Known issues
+
+|<div style="width:250px">Edition|Description|
+|:---|:---|
+|Cloud|Data lake configuration validation is broken in terms of wrong bucket names (AWS S3) and wrong account names (Azure Storage). When saving the settings with an invalid bucket/account name, DataHub fails to quickly detect the problem and will instead run a time-consuming check, which shows up as an ongoing save request in the UI. Eventually the request will fail in the UI with a timeout and the save request in the backend will fail as well. In such a case, please carefully check the bucket/account name and try saving again.|
```
Comment (MWindel) [inline content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md:39]:
```suggestion
|Cloud|Data lake configuration validation is broken in terms of wrong bucket names (AWS S3) and wrong account names (Azure Storage). When saving the settings with an invalid bucket/account name, DataHub fails to quickly detect the problem and will instead run a time-consuming check, which shows up as an ongoing save request in the UI. Eventually, the request will fail in the UI with a timeout and the save request in the back end will fail as well. In such a case, please carefully check the bucket/account name and try saving again.|
```

File: content/release-10-18-0/announcements-10-18-0.md (hunk 3 above)
Comment (MWindel) [reply content/release-10-18-0/announcements-10-18-0.md:216]: @cheinz-sag What is the status on this?

File: content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md (hunk 4 above)
Comment (cheinz-sag) [reply content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md:39]: Comma is fixed; backend is again a standard term in this context.

So all in all we should be done...

File: content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Cumulocity IoT DataHub now uses version 24 of Dremio as query engine. The new version comes along with various features and improvements like a revamped user interface, simplified interaction with PowerBI, improved query performance.
+
+### Efficient ODBC connectivity
+
+The ODBC driver required for running queries against Dremio now relies on Apache Arrow, whose columnar memory format allows for a more efficient ODBC-based query execution. The new driver uses port 32010. The legacy driver continues to work using port 31010.
+
+### Enforcement of single data types
+
+Columns having multiple data types, so-called mixed types, can no more occur. During the offloading process, Cumulocity IoT DataHub takes care that the offloaded data has one type per column. If incoming data might lead to multiple types, either the associated offloading pipeline will be stopped or the resulting schema is automatically evolved.
```
Comment (BeateRixen) [inline content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md:19]:
```suggestion
Columns having multiple data types, so-called mixed types, can no longer occur. During the offloading process, Cumulocity IoT DataHub takes care that the offloaded data has one type per column. If incoming data might lead to multiple types, either the associated offloading pipeline will be stopped or the resulting schema is automatically evolved.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**